Makefile

```diff
@@ -16,7 +18,12 @@ updatedeps:
 
 .PHONY: install
 install:
-	go install ./...
+	go install -ldflags '-X "main.version=dev build $(dev_build_version)"' ./...
+
+.PHONY: release
+release:
+	@GO111MODULE=off go get github.com/goreleaser/goreleaser
```

Comment:
I also added `GO111MODULE=off` here to make sure the `go get` command doesn't mess with the `go.mod` file, otherwise `go get` will add goreleaser and it's dependencies to `go.mod`. There's a chance the behavior of `go get` might change in go1.12 so that we don't have to turn it off because of very active conversation on golang/go#27643.